backend/YouTubeV2/Controllers/PlaylistsController.cs

```diff
@@ -28,9 +28,9 @@ public async Task<ActionResult<CreatePlaylistResponseDto>> CreatePlaylist(Create
         }
 
         [HttpPut("details")]
-        public async Task<ActionResult<UserDto>> UpdatePlaylistDetails([FromQuery][Required] Guid playlistId, PlaylistEditDto request, CancellationToken cancellationToken)
+        public async Task<ActionResult<UserDto>> UpdatePlaylistDetails([FromQuery][Required] Guid id, PlaylistEditDto request, CancellationToken cancellationToken)
         {
-            return Ok(await _playlistsService.UpdatePlaylistDetails(new Guid(GetUserId()), playlistId, request, cancellationToken));
+            return Ok(await _playlistsService.UpdatePlaylistDetails(new Guid(GetUserId()), id, request, cancellationToken));
```

Comment:
why new Guid? if its user id than we are using string no point of creating a guid